source/main.cpp

```diff
@@ -55,7 +56,6 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <string>
 
 #ifdef _WIN32
-#define STRICT
```

Comment:
Why remove?

Reply:
it conflicts with something in the curl header

Reply:
RIP